fe/fe-core/src/main/java/com/starrocks/catalog/OlapTable.java

```diff
@@ -1207,6 +1245,10 @@ public void write(DataOutput out) throws IOException {
             tableProperty.write(out);
         }
 
+        // binlogConfig
+        curBinlogConfig.write(out);
+        out.writeLong(binlogTransactionId);
```

Comment:
I'm afraid that's not downgrade-compatible.
If upgraded to a this version and create the binlog, downgrade to previous version could not recognize this field?